src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -28,7 +28,7 @@ trait AttributeRecord[OWNER_ID] {
   val listLength: Option[Int]
 }
 
-trait AttributeTempRecord[OWNER_ID] extends AttributeRecord[OWNER_ID] {
+trait AttributeScratchRecord[OWNER_ID] extends AttributeRecord[OWNER_ID] {
```

Comment:
I really appreciate the care you took to rename from "temp" to "scratch" which I agree describes this technique accurately.